### DISCUSSION TITLE
Getting error: load library failed: libnvidia-ml.so.1: cannot open shared object file

### DISCUSSION THREAD

**Discussion by Eleksar387** [smy20011/dreambooth-gui#60] [Q&A]
If I hit the train button I get this error message: 

`docker: Error response from daemon: failed to create shim task: OCI runtime create failed: runc create failed: unable to start container process: error during container init: error running hook #0: error running hook: exit status 1, stdout: , stderr: Auto-detected mode as 'legacy'
nvidia-container-cli: initialization error: load library failed: libnvidia-ml.so.1: cannot open shared object file: no such file or directory: unknown.`

Any ideas how to fix this?

**Comment by smy20011** [comment]
Maybe try to make sure your nvidia container toolkit is correctly configured?

https://github.com/NVIDIA/libnvidia-container/issues/135

**Reply by Eleksar387** [reply]
I am using the Windows/WSL 2 version. It doesn't mention that I have to setup anything else.

**Reply by smy20011** [reply]
Did you try to run in inside the WSL2 container? It seems like It running inside of WSL.

**Reply by Eleksar387** [reply]
Puh. I give up. I a just a windows n00b. No idea what I have to do. I remember an older version of dreambooth-GUI once worked. Maybe I have to reinstall Docker and WSL2. Thank you for your help anyway! Keep up the work!
